absl/strings/internal/str_format/checker.h

```diff
@@ -22,7 +22,7 @@
 // Compile time check support for entry points.
 
 #ifndef ABSL_INTERNAL_ENABLE_FORMAT_CHECKER
-#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__)
+#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__) && !defined(__INTELLISENSE__)
```

Comment:
https://google.github.io/styleguide/cppguide.html#Line_Length - Please limit lines to 80 chars.

Reply:
fixed